scripts/dice-stats-main.js

```diff
@@ -327,6 +344,18 @@ class DiceStatsTracker {
             config: true,
             hint: `DICE_STATS_TEXT.settings.${SETTINGS.SHOW_BLIND_ROLLS_IMMEDIATE}.Hint`,
         })
+
```

Comment:
Can Remove Setting As DB interaction is Not auto saved & Loaded anymore instead its up to users to do this